windows/src/global/delphi/cust/CustomisationMessages.pas

```diff
@@ -214,19 +256,27 @@ function TCustomisationMessageManager.MessageFromID(ID: WideString): WideString;
 function TCustomisationMessageManager.MessageFromID(ID,
```

Comment:
I think this could also have same javadoc where it is defined, just to make it quick to understand.